packages/fxa-auth-server/lib/account-events.ts

```diff
@@ -64,6 +78,24 @@ export class AccountEventsManager {
     this.statsd = Container.get(StatsD);
   }
 
+  /**
+   * This function removes and null or undefined values from and object. This ideally
+   * saves of database storage space. This function could be optimized but instead
+   * leaned on the simple approach of using JSON.stringify and JSON.parse because
+   * security events are not blocking events and recorded aysnc.
```

Comment:
Is that true?  I see many awaits below.

Reply:
Hm let me update these to be consistent as well. We don't need the awaits